README.md

```diff
@@ -143,6 +143,86 @@ $event = [
 \VaahEventBrite::organizations()->get();
 ```
 
+
+### VaahStripe
+
+Add Facade in `config/app.php`:
+```php
+'aliases' => [
+...
+'VaahStripe' => WebReinvent\VaahExtend\Facades\VaahStripe::class,
+...
+]
+```
+
+Add env configuration:
+```
+
+...
+STRIPE_API_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+...
+
+```
+
+Reference url: https://stripe.com/docs/api
+
+**Method**
+
+- Stripe One Time Payment
+
+```php
+
+$request = [
+    'customer' => [
+        'name'   => 'xxxxxx',
+        'email'  => 'xx@example.com',
+    ],
+    'payment' => [
+        'currency'      => 'usd',                       // usd / USD
+        'amount'        => '00',
+        'description'   => 'xxxxxx',
+    ],
+    'card' => [
+        'number'        => 'xxxx-xxxx-xxxx-xxxx',    
+        'exp_month'     => '01',                        //01-12
+        'exp_year'      => '2021',
+        'cvc'           => 'xxx',
+    ], 
+    'return_url'   =>  'https://www.google.com/'        // URL to redirect your customer back to after they authenticate or cancel their payment
+];
+
+\VaahStripe::pay($request);
+
+```
+
+- Stripe Subscription
+```php
+
+$request = [
+    'customer' => [
+        'name'   => 'xxxxxx',
+        'email'  => 'xx@example.com',
+    ],
+    'payment' => [
+        'currency'      => 'usd',                       // usd / USD
+        'amount'        => '00',
```

Comment:
@wri-vikram-wri231 why we need to pass the amount, if we are passing package?

Reply:
@themodernpk Sir, If package is not there , then we create package of that amount

Reply:
@wri-vikram-wri231 there should be a completely different method to create package if not exist